src/main/java/io/antmedia/rest/BroadcastRestService.java

```diff
@@ -688,6 +686,39 @@ public Result deleteSubscriber(@ApiParam(value = "the id of the stream", require
 		return new Result(result);	
 	}
 
+	@ApiOperation(value = "Block specific subscriber from playing or publishing.", response = Result.class)
+	@POST
+	@Consumes({ MediaType.APPLICATION_JSON })
+	@Path("/{id}/subscribers/block")
+	@Produces(MediaType.APPLICATION_JSON)
+	public Result blockSubscriber(@ApiParam(value = "the id of the stream", required = true) @PathParam("id") String streamId,
+								  @ApiParam(value = "subscriber block details", required = true) Subscriber subscriber) {
+		boolean result =  false;
+
+		String subscriberId = subscriber.getSubscriberId();
+
+		boolean playBlocked = subscriber.isPlayBlocked();
+		boolean publishBlocked = subscriber.isPublishBlocked();
+
+		long playBlockTime = subscriber.getPlayBlockTime();
+		long playBlockedUntilTime = subscriber.getPlayBlockedUntilTime();
+
+		long publishBlockTime = subscriber.getPublishBlockTime();
+		long publishBlockedUntilTime = subscriber.getPublishBlockedUntilTime();
+
+		if(streamId != null) {
```

Comment:
If the subscriber is publishing or playing and it's not in this instance, we just need to forward the request to the instance that subscriber is active.

Reply:
done